### PR TITLE
10.1 creating comment function on posts

### PR DESCRIPTION
**Pull request by Ephfullstack**

### CODE REVIEW

File: controllers/posts.js [hunk 1]
```diff
@@ -15,11 +15,18 @@ const PostsController = {
   Create: (req, res) => {
     const ObjectId = require("mongodb").ObjectId;
     const id = ObjectId(req.session.user._id);
-    const name = req.session.user.name
-
-    const timePosted = new Date()
-    const post = new Post({userId: id, username: name, message: req.body.message, likes: 0,timestamp: timePosted});
+    console.log(req.body._id);
```
Comment (raphaella-rose) [inline controllers/posts.js:18]: we need to remove this console.log

File: controllers/posts.js [hunk 2]
```diff
@@ -15,11 +15,18 @@ const PostsController = {
...
+    console.log("A");
```
Comment (raphaella-rose) [inline controllers/posts.js:19]: we need to remove this console.log

File: controllers/posts.js [hunk 3]
```diff
@@ -42,14 +49,27 @@ const PostsController = {
...
-    })
-   
-  }
+    });
+  },
+  Comment: (req, res) => {
+    const ObjectId = require("mongodb").ObjectId;
+    const id = ObjectId(req.body.id);
+    console.log(id);
+    console.log("B");
```
Comment (raphaella-rose) [inline controllers/posts.js:63]: we need to remove this console.log

File: controllers/posts.js [hunk 4]
```diff
@@ -42,14 +49,27 @@ const PostsController = {
...
       res.redirect("/posts");
-    })
-   
-  }
+    });
+  },
+  Comment: (req, res) => {
+    const ObjectId = require("mongodb").ObjectId;
+    const id = ObjectId(req.body.id);
+    console.log(id);
```
Comment (raphaella-rose) [inline controllers/posts.js:62]: we need to remove this console.log

File: controllers/posts.js [hunk 5]
```diff
@@ -42,14 +49,27 @@ const PostsController = {
...
+    console.log("B");
+    const comment = req.body.comment;
+    console.log(req.body.comment);
```
Comment (raphaella-rose) [inline controllers/posts.js:65]: we need to remove this console.log

**Review by raphaella-rose — CHANGES_REQUESTED** [review]
This looks good to me, just a couple console.logs to remove. :)

**Review by davekempsell — APPROVED** [review]
Looks good, tests make sense and the code is easy to follow. Just needs the console.logs taken out, then it's all good, so happy to approve

File: controllers/posts.js [hunk 6]
```diff
@@ -15,11 +15,18 @@ const PostsController = {
...
-    const name = req.session.user.name
-
-    const timePosted = new Date()
-    const post = new Post({userId: id, username: name, message: req.body.message, likes: 0,timestamp: timePosted});
+    console.log(req.body._id);
+    console.log("A");
+    const name = req.session.user.name;
 
+    const timePosted = new Date();
+    const post = new Post({
```
Comment (raphaella-rose) [inline controllers/posts.js:23]: does this not need to contain the comments key?

**Review by davekempsell — CHANGES_REQUESTED** [review]
needs console.logs removed, and feature test to be updated to pass

**Pull request closed without merging**